pkg/plugins/init.go

```diff
@@ -0,0 +1,43 @@
+package plugins
+
+import "fmt"
+
+// Plugin register map
+var pluginRegister = map[string]Plugin{}
+
+// Plugin struct to store plugin informations
+type Plugin struct {
+	Template      string
+	DefaultValues map[string]string
+}
+
+// RegisterPlugin to use in CRD file
+func RegisterPlugin(name string, template string, values map[string]string) {
+	pluginRegister[name] = Plugin{Template: template, DefaultValues: values}
+}
+
+// GetDefaultValues get default values by name
+func GetDefaultValues(name string) (map[string]string, error) {
+	var err error
```

Comment:
this can be moved to func GetDefaultValues(name string) `(map[string]string, err error)`